### PR TITLE
Upload page

### PR DESCRIPTION
**Pull request by tanyapowell**
@krissy it's not finished yet BUT for the usability test hopefully it will do...even with all the comments left in lol

### CODE REVIEW

File: app/views/drawings/_form.html.haml [hunk 1]
```diff
@@ -1,4 +1,56 @@
-.drawings-wrapper
+.container
+  %h3 Upload Drawing
+  .row.drawings-wrapper
+    = simple_form_for @drawing, html: { multipart: true } do |f|
+      .col-md-7
+        %h4 Details of Drawing
```
Comment (steverabino) [inline app/views/drawings/_form.html.haml]: add `(required when marked complete)`

File: app/views/drawings/_form.html.haml [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+        %hr
+        .form-group
+          %label{for: "status"} Status (leave this as Pending if you need to enter required details later)
+          = f.input :status, collection: radio_statuses, label_method: lambda {|k| k.last.capitalize }, as: :radio_buttons, required: true, label: false
+      .col-md-5
+        .image-wrap.text-center
+          = image_tag preview_image(@drawing), id: 'image-preview', class: 'img-responsive'
+          %h5 Upload an image (this is required):
+          = f.input :image, label: false, required: true, input_html: { onChange: 'loadFile(event)' }
+          %h5 Details of drawing (required when marked complete)
```
Comment (steverabino) [inline app/views/drawings/_form.html.haml]: This needs deleting (duplicate of line 6)

File: app/views/drawings/_form.html.haml [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+        = f.button :submit, class: 'btn-success btn-block'
+
+:javascript
+  var slider = new Slider('#emoji-slider', {
+  	formatter: function(value) {
+  		return 'Current value: ' + value;
+  	}
+  });
+
+-# .drawings-wrapper
```
Comment (steverabino) [inline app/views/drawings/_form.html.haml]: Is the stuff under here all to be deleted?

Comment (tanyapowell) [inline app/views/drawings/_form.html.haml]: yep, it's to be deleted

**Comment by tanyapowell** [comment]
Will do #87 as a separate PR and will tidy up @stevesmith2609's css hack as a separate PR too

## 

image

**Comment by krissy** [comment]
Awesome work Tan! :+1: There are some responsive bits like the nav bar items disappearing that are existing issues on the live site. We may need to address this as a separate ticket (as this PR is assigned to form page only) before pilot on Tuesday but in a separate branch if they plan to upload images via mobile primarily.

I'm also pushing up a small change to this branch now to make the Edit and Delete icons slightly bigger to address Nina's feedback. Please fix my crappy CSS in a later branch.

**Comment by steverabino** [comment]
:+1: - One thing I've noticed while testing is that when on the mobile size, the upload is underneath all the data entry, which encourages entering all the data first and uploading after.

This is gonna be a pain point if someone enters all the data, and then there's an issue with the upload - we should probably make use of a form that encourages the upload before typing in the data

Again, for another PR!

**Comment by krissy** [comment]
True, if we can quickly switch around image to the left so it's always prioritised at the top is that a quick change we can do tomorrow morning in time to send this over for a pre-pilot test @tanyapowell?

**Comment by tanyapowell** [comment]
Done!

image

**Comment by krissy** [comment]
:+1: :dancers:

**Pull request merged**